rightmesh/contracts/Timelock.sol

```diff
@@ -32,23 +32,30 @@ contract Timelock is Ownable {
   function Timelock(ERC20Basic _token, uint256 _startTime, uint256 _cliffDuration, uint256 _cliffReleasePercent, uint256 _gradualDuration, uint256 _gradualReleasePercentage) public {
 
     // sanity checks
-    assert(_cliffReleasePercent.add(gradualReleasePercentage) <= 100);
+    assert(_cliffReleasePercent.add(_gradualReleasePercentage) <= 100);
     assert(_startTime > now);
```

Comment:
My suggestion is to use "require:, instead of using "assert".

Please refer to docs here:
https://solidity.readthedocs.io/en/develop/control-structures.html#error-handling-assert-require-revert-and-exceptions